bugsnag-android-core/src/main/java/com/bugsnag/android/Event.java

```diff
@@ -84,6 +113,46 @@ public List<Thread> getThreads() {
         return impl.getThreads();
     }
 
+    /**
+     * Create, add and return a new empty {@link Thread} object to this event with a given id
+     * and name. This can be used to augment the report with thread data that would not be picked
+     * up as part of a normal report being generated (for example: native threads managed
```

Comment:
```suggestion
     * and name. This can be used to augment the event with thread data that would not be picked
     * up as part of a normal event being generated (for example: native threads managed
```